packages/ui/src/components/UserWaitingStatus.tsx

```diff
@@ -1,19 +1,34 @@
-import { Heading, Progress, Stack, Text } from "@chakra-ui/react";
+import {
+  Flex,
+  Heading,
+  Progress,
+  Spacer,
+  Stack,
+  Text,
+  useToast,
+} from "@chakra-ui/react";
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { config } from "../lib/config";
 
-export default function UserWaitingwaitingStatus() {
-  const tjoined = new Date("2022-03-10T10:02:00");
-
-  const waitingStatus = {
-    queueName: "QueueA",
-    placeInQ: 5,
-    totPeopleInQ: 30,
-    estimatedwaitingTime: 15,
-    waitingTimeJoined: tjoined.toLocaleString("en-US"),
+export default function UserWaitingStatus(props: { joint: Date }) {
+  const { queueId, userId } = useParams();
+  const tJoined = props.joint;
```

Comment:
Try to avoid single letter names for anything. A variable like timeJoined is much better than tJoined